docs/Makefile

```diff
@@ -1,5 +1,3 @@
-C1541 = c1541
-AS = acme
```

Comment:
Was this change intentional?

Reply:
Keep it simple, you will never assemble in the docs.

Reply:
Ah! I didn't realize it was the docs folder :D